### PR TITLE
PPF-366 - As an admin I can approve an integration

### PR DESCRIPTION
**Pull request by brampauwelyn**
### Added
- Test "as an admin I can approve an `entry-api` integration"
- [extra logic to select different kind of integrations](https://github.com/cultuurnet/publiq-platform/pull/923/commits/cbcd48a823518b907cf6ebbc86f742b6ad58c2ef)

### Changed
- [Use .js imports](https://github.com/cultuurnet/publiq-platform/pull/923/commits/20e0e9ca8014b8868523a6d1014e2d5db8e5cd3a)
- [IntegrationTypes to a types file](https://github.com/cultuurnet/publiq-platform/pull/923/commits/bc25a64988b512b9360d7414d79375f29837b289)
- [Only fill in TVA when IntegrationType is not "entry-api"](https://github.com/cultuurnet/publiq-platform/pull/923/commits/7b57759ef9bdb860534af06d7a68648c123d3a01)

### Removed
- [unused import](https://github.com/cultuurnet/publiq-platform/pull/923/commits/ffcbb2c9a6a3264a9a320bd27ed7102227db4381)

---
Ticket: https://jira.uitdatabank.be/browse/PPF-366

### CODE REVIEW

**Review by LucWollants — APPROVED** [review]

File: e2e/tests/integrations/integrator/request-activation.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  integrationType: IntegrationType
+) {
+  await page.goto(`/nl/integraties/${integrationId}`);
+  await page
+    .getByRole("button", { name: "Activatie aanvragen" })
+    .nth(1)
+    .click();
+  await page
+    .getByLabel("Naam bedrijf, organisatie of")
+    .nth(1)
```
Comment (simon-debruijn) [inline e2e/tests/integrations/integrator/request-activation.ts:17]: Why are there multiple?

File: e2e/tests/types.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export const IntegrationTypes = {
```
Comment (simon-debruijn) [inline e2e/tests/types.ts:1]: It should be possible to include the existing types by changing the tsconfig

**Review by simon-debruijn — APPROVED** [review]
We can change the use of the shared types in a new PR

**Review by simon-debruijn — APPROVED** [review]
Thanks ⭐

**Pull request merged**